tests/ranger/container/test_bookmarks.py

```diff
@@ -56,3 +56,20 @@ def crash():
         secondstore.update_if_outdated()
     secondstore.update = origupdate
     secondstore.update_if_outdated()
+
+
+def test_bookmark_symlink(tmpdir):
+    # Initialize plain file and symlink paths
+    bookmarkfile_link = tmpdir.join("bookmarkfile")
+    bookmarkfile_orig = tmpdir.join("bookmarkfile.org")
+
+    # Create symlink pointing towards the original plain file.
+    os.symlink(str(bookmarkfile_orig), str(bookmarkfile_link))
```

Comment:
Did you consider using `.decode()` instead of `str()`?

Reply:
Not possible unfortunately since `tmpdir` is a fixture from pytest.
It is throwing an AttributeError:
`AttributeError: 'LocalPath' object has no attribute 'decode'`

Reply:
Ah, I thought it was initially a `bytes` object. If that's a path object, that's fine then.